### ISSUE TITLE
wxWidgets port is missing dll files

### ISSUE THREAD

**Issue by vrichomme** [Microsoft/vcpkg#2076]
Hi,

I wanted to build a simple wxWidgets example using VS2015 on Windows 10 64 bits.
I have installed wxWidgets x64 using the cmd: vcpkg.exe install wxwidgets:x64-windows.
When I compile my sample I get the following errors (I have tested in both mbcs and Unicode):

Severity	Code	Description	Project	File	Line	Suppression State
Error	LNK1120	14 unresolved externals	TestWxCustomList	C:\Developer\msys64\home\John\trophy\TestWxCustomList\x64\Debug\TestWxCustomList.exe	1	
Error	LNK2001	unresolved external symbol "bool wxTrapInAssert" (?wxTrapInAssert@@3_NA)	TestWxCustomList	C:\Developer\msys64\home\John\trophy\TestWxCustomList\MainAppFrame.obj	1	
Error	LNK2001	unresolved external symbol "char const * const wxFrameNameStr" (?wxFrameNameStr@@3QBDB)	TestWxCustomList	C:\Developer\msys64\home\John\trophy\TestWxCustomList\MainAppFrame_GUI.obj	1	
Error	LNK2001	unresolved external symbol "char const * const wxStatusLineNameStr" (?wxStatusLineNameStr@@3QBDB)	TestWxCustomList	C:\Developer\msys64\home\John\trophy\TestWxCustomList\MainAppFrame_GUI.obj	1	
Error	LNK2001	unresolved external symbol "class wxEventTypeTag<class wxCommandEvent> const wxEVT_MENU" (?wxEVT_MENU@@3V?$wxEventTypeTag@VwxCommandEvent@@@@B)	TestWxCustomList	C:\Developer\msys64\home\John\trophy\TestWxCustomList\MainAppFrame.obj	1	
Error	LNK2001	unresolved external symbol "class wxEventTypeTag<class wxCommandEvent> const wxEVT_MENU" (?wxEVT_MENU@@3V?$wxEventTypeTag@VwxCommandEvent@@@@B)	TestWxCustomList	C:\Developer\msys64\home\John\trophy\TestWxCustomList\MainAppFrame_GUI.obj	1	
Error	LNK2001	unresolved external symbol "class wxMBConv * wxConvLibcPtr" (?wxConvLibcPtr@@3PEAVwxMBConv@@EA)	TestWxCustomList	

After some search I found on stackoverflow this: https://stackoverflow.com/questions/42036606/wxwidgets-visual-studio-2015-link-errors

and after having added the preprocessor directive WXUSINGDLL I can compile and generate a binary but it doesn't run because the wxWidgets dlls are missing.
So to sum up it seems this wxWidgets port forgot to provide dlls ...

**Comment by vrichomme** [comment]
Besides I cannot find wxpng(d).lib, wxtiff(d).lib, wxzlib(d).lib

**Comment by Type1J** [comment]
I'm not sure about the errors listed above, but there is a problem to where the build doesn't install the .dll files for wxWidgets. They are built, but they're in my `C:\vcpkg\buildtrees\wxwidgets\src\wxWidgets-3.1.0\lib\vc_dllx86-windows-rel` directory (for x86-windows Release). This is the only place in the vcpkg tree that I can find them (They are expected to be in the `C:\vcpkg\installed\x86-windows\bin` directory, or `C:\vcpkg\installed\x86-windows\debug\bin` for debug builds.

**Comment by Cheney-W** [comment]
@vrichomme , Thanks for reporting this issue. 
I can't reproduce it with the latest vcpkg, so I close it first. 
Please open a new issue if this is still a problem for you.
